paddle/phi/kernels/cpu/bitwise_kernel.cc

```diff
@@ -97,3 +122,43 @@ PD_REGISTER_KERNEL(bitwise_not,
                    int16_t,
                    int,
                    int64_t) {}
+
+PD_REGISTER_KERNEL(bitwise_left_shift_arithmetic,
+                   CPU,
+                   ALL_LAYOUT,
+                   phi::BitwiseLeftShiftArithmeticKernel,
+                   uint8_t,
+                   int8_t,
+                   int16_t,
+                   int,
+                   int64_t) {}
+
+PD_REGISTER_KERNEL(bitwise_left_shift_logic,
+                   CPU,
+                   ALL_LAYOUT,
+                   phi::BitwiseLeftShiftLogicKernel,
+                   uint8_t,
+                   int8_t,
+                   int16_t,
+                   int,
+                   int64_t) {}
+
+PD_REGISTER_KERNEL(bitwise_right_shift_arithmetic,
+                   CPU,
+                   ALL_LAYOUT,
+                   phi::BitwiseRightShiftArithmeticKernel,
+                   uint8_t,
+                   int8_t,
+                   int16_t,
+                   int,
+                   int64_t) {}
+
+PD_REGISTER_KERNEL(bitwise_right_shift_logic,
+                   CPU,
+                   ALL_LAYOUT,
+                   phi::BitwiseRightShiftLogicKernel,
+                   uint8_t,
+                   int8_t,
+                   int16_t,
+                   int,
+                   int64_t) {}
```

Comment:
After unification, registration can be reduced to `bitwise_right_shift` and `bitwise_left_shift`